### ISSUE TITLE
Cannot select 00:00 because of the range time, Can it be optional?

### ISSUE THREAD

**Issue by sergiujardan** [ndraiman/ngx-amazing-time-picker#2]
At the moment we cannot selecte 12:00, or 00:00 because of the range time and it will not allow the user to select 00:20 or any time because of the range interval:

 
![image](https://github.com/ndraiman/ngx-amazing-time-picker/assets/10956897/96717811-3436-46fd-8895-cd6d4cbdf309)

rangeTime configuration it should be optional:
`rangeTime: config.rangeTime || {start: '0:0', end: '24:0'},`

**Comment by sergiujardan** [comment]
You'll notice, you cannot select 12. (in your case the style override and you cannot see the disabled button) I will suggest to adjust style as well. 

```scss
button {
       &:disabled {
            color: gray
        }
}
``` 

thanks @ndraiman

**Comment by ndraiman** [comment]
Hi @sergiujardan, I mainly updated this library so I can keep using it in newer angular versions.

From looking at the code, `rangeTime` is optional.

**Comment by sergiujardan** [comment]
Hi @ndraiman, the issue is that the rangeTime is set with default values and that will cause the problem. Try to select 12 hour and you'll notice the issue.

**Comment by ndraiman** [comment]
I fixed a bug in the original library:

Fixed TimePickerComponent.checkDisabled fn not converting string to number, resulting in user unable to pick the 12 hour.

See commit [7520ab6](https://github.com/ndraiman/ngx-amazing-time-picker/commit/7520ab6e45c74276b5bef96e9156a8cfd3e54e62), fixed in v17.0.1

**Comment by sergiujardan** [comment]
Thank you @ndraiman
